src/smirnoffio.py

```diff
@@ -160,7 +166,17 @@ def assign_openff_parameter(ff, new_value, pid):
         else:
             (handler_name, tag_name, value_name, smirks) = pid.split('/')
 
+            from openff.toolkit.typing.engines.smirnoff import ParameterList
```

Comment:
Is ParameterList a new class? It would be nice to have a few lines of comments here because I am confused by this and line 176-178.

Reply:
Ah good catch - this was needed to fix https://github.com/openforcefield/openff-toolkit/issues/884 however now that the fix is merged upstream we should be safe to remove this!